ship.config.js

```diff
@@ -46,16 +47,18 @@ module.exports = {
   pullRequestTeamReviewers: ['instantsearch-for-websites'],
   buildCommand: ({ version }) =>
     `NODE_ENV=production VERSION=${version} yarn build`,
-  // afterPublish: async ({ exec, version }) => {
-  //   await waitUntil(() => {
-  //     return (
-  //       exec(`npm view react-instantsearch@${version}`)
```

Comment:
what was the problem with this that doesn't also exist with latest-version? It also uses the registry

Reply:
TBH, I couldn't figure it out.